gwtp-core/gwtp-crawler-guice/src/main/java/com/gwtplatform/crawler/server/guice/ServiceKey.java

```diff
@@ -14,12 +14,13 @@
  * the License.
  */
 
-package com.gwtplatform.crawler.server;
+package com.gwtplatform.crawler.server.guice;
```

Comment:
What if I just added a @Deprecated and directed them to the new ServiceKey annotation? Rather than bloat the code with unnecessary legacy support.